src/portal/src/i18n/lang/de-de-lang.json

```diff
@@ -0,0 +1,1579 @@
+        "RULE_USER_EXISTING": "Name existiert bereits.",
+        "EMPTY": "Name ist erforderlich",
+        "NONEMPTY": "Darf nicht leer sein",
+        "REPO_TOOLTIP": "Nutzer können in diesem Modus keine Aktivitäten mit den Images durchführen.",
+        "ENDPOINT_FORMAT": "Endknoten muss mit HTTP:// oder HTTPS:// beginnen.",
+        "OIDC_ENDPOIT_FORMAT": "Endknoten muss mit HTTPS:// beginnen.",
+        "OIDC_NAME": "Der Name des OIDC Providers.",
+        "OIDC_ENDPOINT": "Die URL eines OIDC-kompatiblen Servers.",
+        "OIDC_SCOPE": "Der Scope der während der Authentifizierung zum OIDC-Server gesendet wird.  Es muss “openid”, und “offline_access” enthalten. Wenn Google verwendet wird, entfällt “offline_access”.",
+        "OIDC_VERIFYCERT": "Box deaktivieren, falls der OIDC Server mit selbstsignierten Zertifikaten betrieben wird.",
+        "OIDC_GROUP_CLAIM": "Der Name des Claim im ID Token dessen Wert die Liste der Gruppennamen enthält.",
+        "OIDC_GROUP_CLAIM_WARNING": "Es darf ausschließlich Buchstaben, Ziffern, und Unterstriche enthalten. Die Länge muss kürzer als 256 Zeichen sein.",
+        "OIDC_AUTOONBOARD": "Überspringe die Startseite, hierdurch können Nutzer ihren Nutzernamen nicht ändern. Der Nutzername wird aus dem ID Token übernommen.",
+        "OIDC_USER_CLAIM": "Der Name des Claims im ID Token, aus dem der Nutzername stammt. Falls das Feld leer ist, wird 'name' verwendet.",
+        "NEW_SECRET": "Das Secret muss länger als 8 Zeichen sein, mit jeweils einem Groß-, einem Kleinbuchstaben und einer Ziffer."
+    },
+    "PLACEHOLDER": {
+        "CURRENT_PWD": "Aktuelles Passwort eingeben",
+        "NEW_PWD": "Neues Passwort eingeben",
+        "CONFIRM_PWD": "Neues Passwort bestätigen",
+        "USER_NAME": "Nutzernamen eingeben",
+        "MAIL": "Email-Adresse eingeben",
+        "FULL_NAME": "Vollständigen Namen eingeben",
+        "SIGN_IN_NAME": "Nutzername",
+        "SIGN_IN_PWD": "Passwort"
+    },
+    "PROFILE": {
+        "TITLE": "Profil",
+        "USER_NAME": "Nutzername",
+        "EMAIL": "Email",
+        "FULL_NAME": "Vor- und Nachname",
+        "COMMENT": "Kommentar",
+        "PASSWORD": "Passwort",
+        "SAVE_SUCCESS": "Profil erfolgreich gespeichert.",
+        "ADMIN_RENAME_BUTTON": "Nutzername ändern",
+        "ADMIN_RENAME_TIP": "Wähle den Button um den Nutzernamen zu \"admin@harbor.local\" zu ändern. Dies kann nicht rückgängig gemacht werden.",
+        "RENAME_SUCCESS": "Umbenennen erfolgreich!",
+        "RENAME_CONFIRM_INFO": "Warnung, ändern des Namens auf admin@harbor.local, dies kann nicht rückgängig gemacht werden.",
+        "CLI_PASSWORD": "CLI Secret",
+        "CLI_PASSWORD_TIP": "Das CLI Secret kann als Passwort mittels docker/helm cli verwendet werden, um auf Harbor zuzugreifen.",
+        "COPY_SUCCESS": "kopieren erfolgreich",
+        "COPY_ERROR": "kapieren fehlgeschlagen",
+        "ADMIN_CIL_SECRET_BUTTON": "ERZEUGE SECRET",
```

Comment:
English text is also uppercase, since the button-Texts are mostly in uppercase. I will doublecheck that.

also: ADMIN_CIL_SECRET and not ADMIN_CLI_SECRET needs a doublecheck (reminder for me)

Reply:
It's a button, thus I will let it uppercase.
I can't verify this in the GUI, since i don't have a OIDC Provider at hand to use with harbor.
I'm open of moving away from UPPERCASE in Buttons, like other translations, but this requires me to change all Button Texts.